nginx/nginx.conf

```diff
@@ -0,0 +1,77 @@
+user  nginx;
+worker_processes  auto;
+worker_rlimit_nofile 100000;
+
+pid        /var/run/nginx.pid;
+
+events {
+    worker_connections  2000;
+    multi_accept on;
+    use epoll;
+       }
+
+http   {
+    index         index.html index.php;
+    include       /etc/nginx/mime.types;
+    default_type  application/octet-stream;
+    
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" $status $body_bytes_sent "$http_referer" "$http_user_agent" ';
+    log_format  error403  '$remote_addr - [$time_local] "$request"';					  
+    
+    keepalive_timeout   5;
+    autoindex		off;
+    server_tokens       off;
+    port_in_redirect	off;
+    sendfile            on;
+    tcp_nopush          on;
+    tcp_nodelay         on;
+    client_max_body_size 64m;
+    
+    ## Flood protection example (see conf.d/extra_protect.conf)
+    limit_req_zone $binary_remote_addr zone=goeasy:35m rate=1r/s;
```

Comment:
yes, you can control the "burst" at specific location, and maybe add separate zone for that ajax function.